src/test/kotlin/mactest.kt

```diff
@@ -0,0 +1,2 @@
+class mactest {
```

Comment:
You should use [kotlin multiplatform](https://kotlinlang.org/docs/multiplatform.html) (if that's not already the case) so that it runs on every platform easily (linux, windows, mac). You shouldn't create platform-specific tests, but rather "common" tests that run on all platforms.

See https://github.com/ACINQ/bitcoin-kmp for example, have a look at how the project is setup and the kind of tests we created in the `commonTest` directory.